iptables/iptables.go

```diff
@@ -84,7 +84,7 @@ func ConfigureFirewall(firewallConfiguration FirewallConfiguration) error {
 
 	for _, cmd := range commands {
 		if err := executeCommand(firewallConfiguration, cmd, nil); err != nil {
-			log.Error("Aborting firewall configuration")
+			log.Errorf("aborting firewall configuration: %v", err)
```

Comment:
A similar log statement above doesn't print the error, and returns it. _If_ the caller (cobra) outputs the error, it might not be necessary to print it here.